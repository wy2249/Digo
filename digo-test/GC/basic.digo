func digo_main() void {
    a := get_one_str()
    b := get_one_int()
    d1, d2, d3, d4, d5 := get_many()
    p1, p2, p3 := get_three2()
    println("%s %f %d", a + "343221" + p2, p3, p1)
}

func get_one_str() string {
    a := "2394239234"
    if (1 == 1) {
        return "12341235622" + a
    } else if (2==1) {
        return a + "123412334" + "222"
    } else {
        return "!234123"
    }

    return "invalid" + "12343" + "3"
}

func get_three2() (int,string,float) {
    
}


func get_many() (string, string, string, int, string) {
    return "12", "23", "454", 100, "4000"
}

func get_one_int() int {
    return 0
}
