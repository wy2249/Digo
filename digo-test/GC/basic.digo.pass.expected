123412356222394239234343221 0.000000 0
